### ISSUE TITLE
[UX] Updated welcome message to help folks find the Dashboard

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#5624]
## Help first time users find the dashboard quickly.

During a hack-a-thon a while back @jenlampton and I watched a first time contributor struggle to find basic information that is easily available on the dashboard, but it didn't seem to occur to this person that the dashboard might be a good place to get started. This user opened many admin menu items until we finally had to suggest looking at the dashboard.

In response, @jenlampton opened up [issue #4673](https://github.com/backdrop/backdrop-issues/issues/4673) to automatically redirect folks after installation to the dashboard. There were lots of opinions and little progress on that issue. One of the leading arguments (in my opinion) against that suggestion is the idea that immediately after installation a new site owner might rather see the front page of their website, rather than being taken to an administrative page. @klonos suggested a compromise of adding a one time message to the front page that encouraged new site owners to visit the dashboard. 

This seems like an easy change that can be improved upon at any point AND with the new content on the front page, I'm inclined to agree with this idea. 

I'm opening a new issue to focus on this specific solution in it's simplest form. If this idea is adopted, we might either close the other issue or continue to discuss a more complete solution for future releases. 

I suspect that this simple tip could go a long way towards making the first interaction with Backdrop CMS a more pleasant and productive experience for many people.

Here is a PR with my simple proposal.

BEFORE
![image](https://user-images.githubusercontent.com/3144571/167986782-f9898419-7591-4a8b-850e-1860da412239.png)

AFTER:
![image](https://user-images.githubusercontent.com/3144571/167987313-1b49fc4d-4a94-4504-b950-42ac1f49724f.png)

**Comment by stpaultim** [comment]
I've create a PR. It can't really be tested in the Sandbox, because the message only appears immediately after installation. You can test it by either applying this PR to a fresh copy of codebase and going through install process - or in this case, it might be sufficient to simply review the screenshot provided. 

I think it's pretty clear from the code, that this is a one line PR that simply adds some text.

**Comment by stpaultim** [comment]
@indigoxela pointed out that the additional message will only work with the standard profile. I've added a check to verify that it's using the standard profile. 

While I understand that it's possible to launch Backdrop with something other than the standard profile, I don't know how to do it. I'd like to test this without the standard profile, how do I do that?

**Comment by indigoxela** [comment]
> I don't know how to do it

There are two options:

If there's an additional profile in the core/profiles/ directory, you'd get a dropdown select list to choose via the web installer.

If not, you can use the command line install script, which also has a "profile" argument to choose the hidden "minimal" profile.
For that make core/scripts/install.sh executable (with chmod).
Then run `./core/scripts/install.sh minimal --db-url=mysql://root:pass@localhost:port/dbname ... // more options here`
See `./core/scripts/install.sh --help for more info.

**Comment by indigoxela** [comment]
Oh, wait, an easier option would be to remove the 'hidden = TRUE' from the minimal profile (core/profiles/minimal/minimal.info). Then you should be able to choose that via the installer UI.

**Comment by stpaultim** [comment]
@indigoxela - thanks for the help here and thanks to @BWPanda for answering in the forum (I posted the question there to better document the answer). 

https://forum.backdropcms.org/forum/how-use-custom-install-profile-backdrop#comment-4049

I just tested with the minimal install profile and as expected, this is what it looks like.

![image](https://user-images.githubusercontent.com/3144571/168017850-3f623ab6-c63b-4eb4-a66a-421815bc5da6.png)

**Comment by BWPanda** [comment]
@stpaultim Instead of checking for the Standard profile, I think it'd be better to check for the Dashboard module. Then even if Dashboard is removed from Standard this won't break, and it'll also work for other profile that include the Dashboard.

**Comment by stpaultim** [comment]
@BWPanda Good idea. I'll work on that ASAP.

**Comment by stpaultim** [comment]
![image](https://user-images.githubusercontent.com/3144571/168398585-fc585ed0-5290-4d2e-a482-fb04bd924a41.png)

**Comment by BWPanda** [comment]
Requested some changes. Don't like the paragraphs tags, perhaps breaks instead...

**Comment by stpaultim** [comment]
> Requested some changes. Don't like the paragraphs tags, perhaps breaks instead...

Yeah, I thought that might be a problem. But, wasn't sure that breaks would be any better. Happy to try that.

**Comment by stpaultim** [comment]
A single `<br>` leaves the lines close together like this. It would seem weird to simply do two `<br><br>` next to each other. 

On the other hand, I like this. I also removed the word `the` from the link. That seemed a little weird and was a holdover from earlier PR.

**Comment by BWPanda** [comment]
I like it! Code looks good to me, so marking RTBC (since there's no separate 'reviewed' tag), but I haven't actually tested.

**Comment by BWPanda** [comment]
Or has been updated and looks good. I was considering suggesting checking if the current user has access to the Dashboard before showing the link, but then I guess the message is only shown on first install in which case the user is an admin with all permissions...

**Comment by jenlampton** [comment]
I just had [the same circular thought process](https://github.com/backdrop/backdrop/pull/4073)  as @BWPanda when reviewing the PR :) This looks good to me as-is, RTBC.

**Comment by stpaultim** [comment]
@jenlampton - What about the issue that @quicksketch raised about using the Distribution name?

**Comment by quicksketch** [comment]
The PR at https://github.com/backdrop/backdrop/pull/4073 has been updated to address using the distribution name as a variable in the new message. Looks good to me.

**Comment by quicksketch** [comment]
Merged https://github.com/backdrop/backdrop/pull/4073 into 1.x and 1.23.x. Thanks @jenlampton, @indigoxela, @BWPanda, and @stpaultim!

**Comment by stpaultim** [comment]
WOW, didn't expect this to get in today. Thanks.

**Comment by klonos** [comment]
Sorry for coming late to the party (haven't been contributing as much lately). A couple of comments/thoughts:

- If we added the additional message with a second `backdrop_set_message()` inside the `if (!backdrop_is_cli())` conditional, then there wouldn't be a need for that `'<br>'` we added - the "Welcome" message and the "might wanna check out the Dashboard" one would have been 2 separate bullet points in that status message shown.
- Instead of adding the message in `install.core.inc` and having to check for `if (module_exists('dashboard'))`, we could have added it in the Dashboard module itself, with a `hook_user_login()` implementation. That way, we could check if the user logs in for the first time, and then show this message to all "regular" users that have permission to use the Dashboard the first time they log into the site (these are the people that need this information the most) - not just user1 after they have finished installing Backdrop.

Separate follow-up issue, or reopen this with a new PR?
